samples/subsys/video/capture/src/main.c

```diff
@@ -106,7 +106,7 @@ int main(void)
 			return 0;
 		}
 
-		printk("\rGot frame %u! size: %u; timestamp %u ms", frame++, vbuf->bytesused,
+		printk("\rGot frame %u! size: %u; timestamp %u ms\n", frame++, vbuf->bytesused,
```

Comment:
Hmmm. I think based on the `\r` at the start of this print statement, the choice to replace the current line with a new one was intentional. I'm ok to make this change, but we should drop the `\r` if we are changing this

Reply:
Yes, for me, `\r` is working on Teraterm but not on Serial Monitor of VSCode. "\n" is working for both platform. And by keeping the logs (with \n), we can look at the timestamps to compute the frame rate which to me, is better.

BTW, as you said, \r should be removed.